upbound_uxp/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# CHANGELOG - Upbound UXP
+
+## 1.0.0 / 2023-07-01
+
+* [Added] Initial Release
```

Comment:
This was a recent change, but could you adjust this to be:

```suggestion
***Added***

* Initial Release
```

This will adhere to the recent formatting change that CHANGELOGs have undergone.

Reply:
Updated